### PR TITLE
zh-translation:/docs/tasks/traffic-management/mirroring/index.md

### PR DESCRIPTION
**Pull request by csdnshyang**
- ref: servicemesher/istio-official-translation#1124

### CODE REVIEW

**Comment by csdnshyang** [comment]
@rootsongjc

File: content/zh/docs/tasks/traffic-management/mirroring/index.md [hunk 1]
```diff
@@ -1,26 +1,21 @@
...
-In this task, you will first force all traffic to `v1` of a test service. Then,
-you will apply a rule to mirror a portion of traffic to `v2`.
+在此任务中，首先把流量全部路由到 `v1` 版本的测试服务。然后，执行规则将一部分流量镜像到 `v2` 版本。
 
-## Before you begin
+## 开始之前{#before-you-begin}
 
-* Set up Istio by following the instructions in the
-  [Installation guide](/docs/setup/).
+* 按照[安装指南](/docs/setup/)中的说明设置 Istio。
```
Comment (rootsongjc) [inline content/zh/docs/tasks/traffic-management/mirroring/index.md]: 请在链接前增加中文前缀 `/zh`。

Comment (csdnshyang) [reply content/zh/docs/tasks/traffic-management/mirroring/index.md]: fixed.

**Pull request merged**